lib/alerts_viewer_web/components/core_components.ex

```diff
@@ -570,7 +575,11 @@ defmodule AlertsViewerWeb.CoreComponents do
             >
               <div class="block py-4 pr-6">
                 <span class="absolute -inset-y-px right-0 -left-4 group-hover:bg-zinc-50 sm:rounded-l-xl" />
-                <span class={["relative", i == 0 && "font-semibold text-zinc-900"]}>
+                <span class={[
+                  "relative",
+                  i == 0 && "font-semibold text-zinc-900",
+                  Map.get(row, :row_class, "")
```

Comment:
~~suggestion: Do we need to define a component `attr` for `row_class`?~~
Nevermind. I see now that it's not really an attr. I wish there was a cleaner way to accomplish this, but this seems good enough for now.

Reply:
Yeah, I don't love this, if we end up doing more stuff per row we'll come up with something better.